pandas/_libs/tslib.pyx

```diff
@@ -58,6 +57,9 @@ from datetime cimport (
 from datetime import timedelta, datetime
 from datetime import time as datetime_time
 
+from tslibs.np_datetime cimport check_dts_bounds as _check_dts_bounds
```

Comment:
is there a reason for the rename?

Reply:
... because you told me to change it

Reply:
the de-privatize is what I am referring. I don't recall asking for that and its not necessary.

Reply:
Oh.  De-privatized because it isn't actually private when is this module.  But I'll revert it anyway.

Reply:
de-privatize here (you did elsewhere)